layers/+tools/xclipboard/packages.el

```diff
@@ -13,8 +13,20 @@
 (defconst xclipboard-packages
   '(
     (spacemacs-xclipboard :location local)
+    (cliphist :requires ivy
+              :toggle xclipboard-enable-cliphist)
     ))
 
+(defun xclipboard/init-cliphist ()
+  (use-package cliphist
+    :init (spacemacs/set-leader-keys
+            "xP" 'cliphist-paste-item
+            "xR" (defun spacemacs/xclipboard-cliphist-paste-item-rectangle ()
```

Comment:
Create a new `xclipboard/funcs.el` file with this header:
```
;;; funcs.el --- xclipboard layer functions file for Spacemacs.
;;
;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
;;
;; Author: <author>
;; URL: https://github.com/syl20bnr/spacemacs
;;
;; This file is not part of GNU Emacs.
;;
;;; License: GPLv3
```
Replace `<author>` in the header with your name.
The xclipboard layers author included their email in the `packages.el` header.
https://github.com/syl20bnr/spacemacs/blob/9637ceb389a3db0450cec3831a38d4ab417a5912/layers/%2Btools/xclipboard/packages.el#L5
But it's up to you if you want to include yours.


Then add the function definition from the `xR` key binding, to the new `funcs.el` file, and have `xR` call the `'spacemacs/xclipboard-cliphist-paste-item-rectangle` function.

Reply:
Updated.